### ISSUE TITLE
A line in bed file gives a segment fault error

### ISSUE THREAD

**Issue by rraadd88** [arq5x/bedtools2#642]
Dear developer,
I am using `bedtools intersect` (v2.25.0) to get annotations for my bed file.
Here's the command I am using

```
bedtools intersect -wa -wb -loj -a sorted.bed -b Homo_sapiens.GRCh38.92.gff3.gz.sorted.gff3.gz > annotations.bed
```
I get this error
```
Segmentation fault (core dumped)
bash command error: 139
```
for this line in bed file

Y | 63976 | 63999 | Y\|+63977\|23M\|4 | 4 | +
-- | -- | -- | -- | -- | --

but no error for this line

Y | 25409994 | 25410017 | Y\|-25409995\|23M\|5 | 5 | -
-- | -- | -- | -- | -- | --

Please let me know what's off here.

******
Here's a set of bash commands to recreate this error:
```
#get gff
wget ftp://ftp.ensembl.org/pub/release-92/gff3/homo_sapiens/Homo_sapiens.GRCh38.92.gff3.gz
#sort gff
bedtools sort -i Homo_sapiens.GRCh38.92.gff3.gz > Homo_sapiens.GRCh38.92.gff3.gz.sorted.gz
# bed file that gives error
echo -e 'Y\t63976\t63999\tY|+63977|23M|4\t4\t+' > err.bed
# bed file that does not give error
echo -e 'Y\t25409994\t25410017\tY|-25409995|23M|5\t5\t-' > noerr.bed

# test
bedtools intersect -wa -wb -loj -a err.bed -b Homo_sapiens.GRCh38.92.gff3.gz.sorted.gz 
# Segmentation fault (core dumped)

bedtools intersect -wa -wb -loj -a noerr.bed -b Homo_sapiens.GRCh38.92.gff3.gz.sorted.gz 
# Y	25409994	25410017	Y|-25409995|23M|5	5	-	Y	Ensembl	chromosome	2781480	56887902	.	.	.	ID=chromosome:Y;Alias=CM000686.2,chrY,NC_000024.10

```

**Comment by jmarshall** [comment]
This was fixed by 9b734e634e3f42784099d2cd77177a90fa4e5f37; releases 2.26.0 and afterwards contain the fix.

**Comment by rraadd88** [comment]
Thank you very much. Issue solved.
